models.py

```diff
@@ -396,6 +400,13 @@ def sample(
         tqdm_pbar: Optional[Callable] = None,
         return_last: bool = True,
         return_aux: bool = False,
+        use_superposition: bool = True,
```

Comment:
Can we move these above the `noise_schedule` argument? that way we can keep them closer to the other sampling hyperparameters. Also, please add these new arguments to the docstring (no need for the comments indicating when they were added.)